ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -706,9 +706,9 @@ periodics:
   cron: '@yearly'
   decorate: true
   labels:
-    job-env: metal
+    job-env: metal-ipi
```

Comment:
If a job is a default -- IPI, OpenshiftSDN, parallel test suite, etc we normally don't add qualifiers.  e.g. e2e-aws implies ipv4, ipi, openshiftsdn.  

I realize the e2e-metal-ipi jobs are named this way since UPI metal took the name, but that really should get fixed at some point.  If we can avoid carrying that `ipi` suffix into cluster-bot, that'd be great.

Reply:
Changed to `metal`. Thanks.